package.json

```diff
@@ -46,9 +46,12 @@
     "async": "^2.5.0",
     "axe-core": "^2.6.1",
     "chai": "^4.1.2",
+    "express": "^4.16.2",
```

Comment:
Should these be peer dependencies since some people will not need/want to use the Express server?

Reply:
If these are moved to peerDeps, they should also be added as devDeps so we can still test toolkit with them.

Reply:
The current toolkit setup is listing the Service dependencies under the peer dependencies and then describing the dependency requirements in the read-me sections. see https://github.com/cerner/terra-toolkit/blob/master/docs/Nightwatch_Utility.md#getting-started

Reply:
I think the current setup is incorrect. We should not be using peer dependencies as 'optional' dependancies. The peer dependency warnings produced do not indicate that this is an 'optional' dependency. terra-toolkit should be added as a dev dependency to any project consuming it. The only harm that could be caused by including express would be a larger node_modules folder in the dev env. The delivered package size to the client won't be increased, nor the production server size. For terra-toolkit to work properly we should add any dependancies that are needed for it's components to work as dependencies. If we feel strongly about not having this service available by default we should split it out into it's own package. I don't think that is necessary.

Reply:
Fair enough. +1